CONTRIBUTING.md

```diff
@@ -115,4 +115,21 @@ $ node_modules/.bin/gulp serve --server=<che_api_url>
 
 
 ### Che server a.k.a WS master
-To be provided soon.
+There is a [devfile](https://github.com/eclipse/che/blob/master/devfile.yaml) for development of Che server in Che.
+To build Che one may run a predefined build task from the devfile.
+
+Starting Che master requires some manual steps:
```

Comment:
Apply, please, information that che should starts from `che-server-runtime` container.

Reply:
Done